views/index.html

```diff
@@ -64,7 +64,7 @@ <h3 class="release--title">
 
 {% for type, items in apis %} {% if items.length %}
 <section>
-  <h2>{{sections[type] | safe}}</h2>
+  <h2 id="{{sections[type].split(' ').slice(0, 2).join('-') | slug}}">{{sections[type] | safe}} <a href="#{{sections[type].split(' ').slice(0, 2).join('-') | slug}}">#</a></h2>
```

Comment:
```suggestion
  {% set sectionID = "{{sections[type].split(' ').slice(0, 2).join('-') | slug}}" %}
  <h2 id="{{sectionID}}">{{sections[type] | safe}} <a href="#{{sectionID}}">#</a></h2>
```

Because it's being used in two places, we can use Nunjuck's [set](https://mozilla.github.io/nunjucks/templating.html#set) functionality to make sure we've got the same thing in both places by making an on-the-fly variable.

Reply:
Hey Tom, thanks for taking this PR back down to just section links. Can you please update this line so we're not calculating the ID twice? It'll make this easier to read. Thanks!